Dockerfile

```diff
@@ -2,7 +2,7 @@ ARG GIT_COMMIT=unspecified
 ARG GIT_REMOTE=unspecified
 ARG VERSION=unspecified
 
-FROM python:3.7-alpine
+FROM python:3.9.0-alpine
```

Comment:
So... do we go with `3.9.0-alpine` or `3.9-alpine`?

Reply:
I vote for `3.9-alpine` so we always get the latest 3.9 version with Alpine Linux and Python 3.9.